src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -321,12 +321,17 @@ def load_migration_products_offline(activations)
 
         return :empty unless migration_confirmed?(base_product, activations)
 
+        # evaluate pkg.product for version_version
+        pkg_product = Yast::Pkg.ResolvableProperties(base_product.name,
+          :product, base_product.version).find_all.first
+
         remote_product = OpenStruct.new(
-          arch:         base_product.arch.to_s,
-          identifier:   base_product.name,
-          version:      base_product.version,
+          arch:            base_product.arch.to_s,
+          identifier:      base_product.name,
+          version:         base_product.version,
```

Comment:
Could we directly set the correct "version" here instead of having extra "version_version"?

If it is really needed then add a comment why we need the extra value.